etc/eve7/GL.controller.js

```diff
@@ -104,7 +104,7 @@ sap.ui.define([
       fShapeId: 256, fShapeBits: 1024, fDX: 200, fDY: 300, fDZ: 400, fOrigin: [0,0,0]
       };
 
-      var geom_obj = JSROOT.extend(JSROOT.Create("TEveGeoShapeExtract"),
+      var geom_obj = JSROOT.extend(JSROOT.Create("REveGeoShapeExtract"),
                       { fTrans: null, fShape: shape, fRGBA: [0, 1, 0, 0.2], fElements: null, fRnrSelf: true });
```

Comment:
TEveGeoShapeExtract was not renamed.
I could fix it myself after merge

Reply:
Ah, good ... yes, please.